src/transformers/image_utils.py

```diff
@@ -184,27 +184,64 @@ def normalize(self, image, mean, std):
         else:
             return (image - mean) / std
 
-    def resize(self, image, size, resample=PIL.Image.BILINEAR):
+    def resize(self, image, size, resample=PIL.Image.BILINEAR, torch_resize=False, max_size=None):
         """
         Resizes `image`. Note that this will trigger a conversion of `image` to a PIL Image.
 
         Args:
             image (`PIL.Image.Image` or `np.ndarray` or `torch.Tensor`):
                 The image to resize.
             size (`int` or `Tuple[int, int]`):
-                The size to use for resizing the image.
+                The size to use for resizing the image. If `size` is a sequence like (h, w), output size will be
+                matched to this.
+
+                If `size` is an int and `torch_resize` is `False`, then image will be resized to (size, size). If
+                `size` is an int and `torch_resize` is `True`, then smaller edge of the image will be matched to this
+                number. i.e, if height > width, then image will be rescaled to (size * height / width, size).
             resample (`int`, *optional*, defaults to `PIL.Image.BILINEAR`):
                 The filter to user for resampling.
+            torch_resize (`bool`, *optional*, defaults to `False`):
+                Whether or not to replicate `torchvision.transforms.Resize` with support for resizing only the smallest
+                edge and providing an optional `max_size`.
```

Comment:
And here we would say: `torchvision.transforms.Resize` (with a link) uses the xxx value.